.github/CODEOWNERS

```diff
@@ -1 +1,4 @@
-*	@BupycHuk
+*                           @BupycHuk
+/managed/                   @BupycHuk @artemgavrilov
```

Comment:
Is there any benefit in adding multiple entries here at the moment? We have only 2 people responsible more or less for the entire codebase. It would make more sense if we had more people looking at specific parts of the project.

Reply:
Right now we just moved what we had in managed repo. But yes, we will add more codeowners for specific parts here